### PR TITLE
3 get api categories

### PR DESCRIPTION
**Pull request by 16000psi**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        .expect(200)
+        .then(({body}) => {
+            expect(Array.isArray (body)).toBeTruthy()
+        })
+    })
+    test("Array should contain objects with slug and description keys", () => {
+        return request(app)
+        .get("/api/categories")
+        .then(({body}) => {
+            for (let i in body) {
```
Comment (Augs0) [inline __tests__/app.test.js:36]: If iterating over an array, we should check its length first so we know 1. that we have something to iterate over 2. that we have the right amount of elements that we expect

File: app.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+const { handle500Error, handleCustomErrors, getCategories } = require("./controllers/index");
+
+// console.log(getCategories)
```
Comment (Augs0) [inline app.js:4]: Don't forget to remove any console logs or comments before pushing code

File: app.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+const app = express();
+
+app.use(express.json());
```
Comment (Augs0) [inline app.js:8]: This shouldn't be here yet as we aren't posting yet

File: controllers/get-api-categories.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+const categories = require("../db/data/test-data/categories")
+const fetchCategories = require ("./../models/fetch-categories")
+
+function getCategories  (req, res, next) {
+
+    fetchCategories()
+    .then((categories) => {
+        res.status(200).send(categories)
```
Comment (Augs0) [inline controllers/get-api-categories.js:8]: We should ideally send our data on a key so we can easily find it on the other end. We just need some curly braces around categories here to do that

**Review by Augs0 — CHANGES_REQUESTED** [review]
Just a few small changes before you move on, but well done so far :D

File: controllers/get-api-categories.js (hunk 4 above)
Comment (16000psi) [reply controllers/get-api-categories.js:8]: Awesome thanks August! :)

**Pull request merged**